### PR TITLE
[#130] Add information about migrating from legacy hamster

### PR DESCRIPTION
**Pull request by elbenfreund**
Closes: #130

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -73,6 +73,30 @@ as well (we still run the 'style checks' at least).
 We hope to get to the bottom of this at some point and would be most grateful
 if you have any hint or pointer that may help tracking down this issue.
 
+Migrating from 'legacy hamster'
+---------------------------------
+In case you are wondering "Will I be able to continue to use my 'legacy hamster'
+database with this rewrite?" the answer is "yes and no".
+This new version of hamster significantly raises the standart in terms of data consistency. Unlike
```
Comment (jtojnar) [inline README.rst]: standard

File: README.rst [hunk 2]
```diff
@@ -73,6 +73,30 @@ as well (we still run the 'style checks' at least).
...
+before there will be no way to have "Facts" that do not have a dedicated "end" or even
+"multiple facts at a given point in time".
+
+There will be a way to import data that still constitutes valid 'facts' (that
```
Comment (jtojnar) [inline README.rst]: constitute

File: README.rst [hunk 3]
```diff
@@ -73,6 +73,30 @@ as well (we still run the 'style checks' at least).
...
+is has a start and end time). We have however not decided on how this will be
+implemented nor what to do with legacy 'facts' that do not have an end time.
+
+The general timeline for adressing the actual implementation is: once we are feature freezing
+in preparation of release 1.0.0 as part of a more general pre-release cleanup effort.
+
+Whilst possible, it is unlikely we will have the resources to provide a fancy looking GUI to
+resolve migration conflicts (unless someone new pitches is of cause) so the result will most
```
Comment (jtojnar) [inline README.rst]: replace is with comma, cause → course

File: README.rst [hunk 4]
```diff
@@ -73,6 +73,30 @@ as well (we still run the 'style checks' at least).
...
+likely a migration script of some sort.
```
Comment (jtojnar) [inline README.rst]: add “be”

**Review by jtojnar — CHANGES_REQUESTED** [review]

**Comment by jtojnar** [comment]
Additionally, I feel like there are some commas missing .

**Comment by elbenfreund** [comment]
Done

File: README.rst (hunk 2 above)
Comment (elbenfreund) [reply README.rst]: hm, this actually looks right to me. Can you elaborate?

Comment (jtojnar) [reply README.rst]: I try not to be a grammar nazi and accept that “data” can be sometimes be uncountable but IMHO when the word can be substituted for a plural word like facts or items, it should be regarded as a plural noun.

**Review by jtojnar — APPROVED** [review]

**Pull request merged**